contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+
+        if(msg.value > challengeReward) msg.sender.transfer(msg.value - challengeReward); // Refund any extra eth.
+        Agreement memory agreement = agreements[latestAgreementId(_tokenID)];
+        emit ItemStatusChange(agreement.parties[0], agreement.parties[1], _tokenID, item.status, agreement.disputed);
+    }
+
+    /**
+     *  @dev Request an item to be cleared.
+     *  @param _tokenID The tokenID of the item to clear.
+     *  @param _metaEvidence The meta evidence for the potential dispute.
+     *  @param _arbitrationFeesWaitingTime The maximum time to wait for arbitration fees if the dispute is raised.
+     *  @param _arbitrator The arbitrator to use for the potential dispute.
+     */
+    function requestClearing(
+        bytes32 _tokenID,
+        string _metaEvidence,
+        uint _arbitrationFeesWaitingTime,
+        Arbitrator _arbitrator
+    ) public payable {
+        Item storage item = items[_tokenID];
+        Agreement memory prevAgreement = agreements[latestAgreementId(_tokenID)];
+        require(!prevAgreement.disputed || prevAgreement.executed, "There is already a request in place");
+        require(msg.value >= challengeReward, "Not enough ETH.");
+
+        if (item.status == ItemStatus.Registered)
+            item.status = ItemStatus.ClearingRequested;
+        else if (item.status == ItemStatus.Absent)
+            item.status = ItemStatus.PreventiveClearingRequested;
+        else
+            revert("Item in wrong status for clearing."); // If the item is neither Registered nor Absent, it is not possible to request clearing it.
+
+        if (item.lastAction == 0) {
+            itemsList.push(_tokenID);
+        }
+
+        item.balance += challengeReward;
+        item.lastAction = now;
+        item.challengeReward = challengeReward;
+
+        address[] memory _parties = new address[](2);
+        _parties[0] = msg.sender;
+
+        _createAgreement(
+            _tokenID,
+            _metaEvidence,
+            _parties,
+            2,
+            new bytes(0),
+            _arbitrationFeesWaitingTime,
+            _arbitrator
+        );
+
+        if(msg.value > challengeReward) msg.sender.transfer(msg.value - challengeReward); // Refund any extra eth.
+
+        Agreement memory agreement = agreements[latestAgreementId(_tokenID)];
+        emit ItemStatusChange(agreement.parties[0], agreement.parties[0], _tokenID, item.status, agreement.disputed);
+    }
+
+    /** @dev Overrides parent to use information specific to Arbitrable Token List in math:
+     *  - Parent's fundDispute doesn't take into account `challengeReward` when calculating eth.
+     *  For calls that initiate a dispute, msg.value must also include `challengeReward`.
+     *  @param _agreementID The ID of the agreement.
+     *  @param _side The side. 0 for the side that lost the previous round, if any, and 1 for the one that won.
+     */
+    function fundDispute(bytes32 _agreementID, uint _side) public payable {
```

Comment:
I've made this function external too.

Reply:
You can't change the visibility modifier in an override.